cvat-ui/src/reducers/annotation-reducer.ts

```diff
@@ -947,10 +948,19 @@ export default (state = defaultState, action: AnyAction): AnnotationState => {
         }
         case AnnotationActionTypes.CHANGE_ANNOTATIONS_FILTERS: {
             const { filters } = action.payload;
+            const { filtersHistory, filters: oldFilters } = state.annotations;
```

Comment:
Actually why do you need oldFilters here?
All old filters already should be in filtersHistory (after previous ``CHANGE_ANNOTATIONS_FILTERS`` action). Am I wrong?

Reply:
@bsekachev if there are more than 10 active filters for example.

Reply:
@bsekachev, I guess it's not so good
![image](https://user-images.githubusercontent.com/8168727/75781542-717ca000-5d6e-11ea-88fa-64520ff1d2df.png)